optimum/exporters/tasks.py

```diff
@@ -189,7 +189,9 @@ class TasksManager:
 
         _DIFFUSERS_TASKS_TO_MODEL_LOADERS = {
             "stable-diffusion": "StableDiffusionPipeline",
-            "stable-diffusion-xl": "StableDiffusionXLImg2ImgPipeline",
+            "stable-diffusion-xl": "StableDiffusionXLPipeline",
+            "stable-diffusion-xl-refiner": "StableDiffusionXLImg2ImgPipeline",
+            "latent-consistency": "LatentConsistencyModelPipeline",
```

Comment:
Could you extend on why this is needed ? To my knowledge this shouldn't be needed (both pipelines can be correctly loaded using the current `_DIFFUSERS_TASKS_TO_MODEL_LOADERS`

Reply:
also at the moment inferred task for `diffusers` models will either be "stable-diffusion-xl" or "stable-diffusion" https://github.com/huggingface/optimum/blob/dedb852eaea3d3d899920f4017b9fd899eb2756b/optimum/exporters/tasks.py#L1564

Reply:
This is necessary to correctly determine the model class from task. Hybrid quantization supports the ` stable-diffusion-xl` task, but doesn't support the `stable-diffusion-xl-refiner`. Without these changes, it is not possible to apply hybrid quantization via CLI.

Reply:
Not sure that we need it in the TaskManager, from what I see it's only needed to get the class to load the model in the optimum-intel CLI for diffusers models, could we move it in optimum-intel directly instead ? We could get this information by combining the task (text-to-image, image-to-image,  inpaint, ..) and the architecture (SD, SDXL, LCMs, ..) using the model's config

Reply:
Are you suggesting to remove only the `latent-consistency` from `_DIFFUSERS_TASKS_TO_MODEL_LOADERS`?

As far as I understand, there are no `text-to-image, image-to-image, inpaint,..` tasks in optimum. Should I create these tasks only for optimum-intel instead of `stable-diffusion, stable-diffusion-xl, stable-diffusion-xl-refiner and latent-consistency`? In this case, we cann't use `TasksManager.infer_task_from_model` for diffusion models.